tasks/lib/postcss-notnested.js

```diff
@@ -0,0 +1,46 @@
+var postcss = require('postcss');
+
+module.exports = postcss.plugin('postcss-notnested', function (opts) {
+  opts = opts || {};
+
+  // Match ampersands at the start of a given selector
+  var re = /^&.*/;
+
+  return function (root, result) {
+    root.walkRules((rule, ruleIndex) => {
+      if (rule.selectors) {
+        if (opts.replace) {
+          var replaced = false;
+          var selectors = rule.selectors.map(selector => {
+            if (re.test(selector)) {
+              replaced = true;
+              return selector.replace(re, opts.replace);
+            }
+            else {
+              return selector;
+            }
+          });
+
+          if (replaced) {
+            rule.selectors = selectors;
+          }
+        }
+        else {
```

Comment:
Otherwise, all selectors that use `&` are killed, and if no selectors remain, the entire rule is killed.